src/backend/TrafficCourts/Workflow.Service/appsettings.Development.json

```diff
@@ -7,7 +7,7 @@
     "BaseUrl": "http://localhost:5010/"
   },
   "SmtpConfiguration": {
-    "Host": "localhost",
+    "Host": "apps.smtp.gov.bc.ca",
```

Comment:
I don't know if this should be checked in.  Maybe this should be part of UserSecrets?

For other projects, I've used gessnerfl/fake-smtp-server, a docker image that can be run within docker-compose and simulates an email server. All it does is just log the emails that were sent, but in a nice HTML website so we can see how emails with HTML content are rendered.

Reply:
i put this in user secrets